docs/getting-started-guides/docker-multinode.md

```diff
@@ -0,0 +1,261 @@
+```--iptables=false``` so that it can only run containers with ```--net=host```.  That's sufficient to bootstrap our system.
+
+Run:
+```sh
+sudo sh -c 'docker -d -H unix:///var/run/docker-bootstrap.sock -p /var/run/docker-bootstrap.pid --iptables=false --ip-masq=false --bridge=none --graph=/var/lib/docker-bootstrap 2> /var/log/docker-bootstrap.log 1> /dev/null &'
+```
+
+### Startup etcd for flannel and the API server to use
+Run:
+```
+sudo docker -H unix:///var/run/docker-bootstrap.sock run --net=host -d kubernetes/etcd:2.0.5.1 /usr/local/bin/etcd --addr=127.0.0.1:4001 --bind-addr=0.0.0.0:4001 --data-dir=/var/etcd/data
+```
+
+Next, you need to set a CIDR range for flannel.  This CIDR should be chosen to be non-overlapping with any existing network you are using:
+
+```sh
+sudo docker -H unix:///var/run/docker-bootstrap.sock run --net=host kubernetes/etcd:2.0.5.1 etcdctl set /coreos.com/network/config '{ "Network": "10.1.0.0/16" }'
+```
+
+
+### Set up Flannel on the master node
+Flannel is a network abstraction layer build by CoreOS, we will use it to provide simplfied networking between our Pods of containers.
+
+Flannel re-configures the bridge that Docker uses for networking.  As a result we need to stop Docker, reconfigure its networking, and then restart Docker.
+
+#### Bring down Docker
+To re-configure Docker to use flannel, we need to take docker down, run flannel and then restart Docker.
+
+Turning down Docker is system dependent, it may be:
+
+```sh
+sudo /etc/init.d/docker stop
+```
+
+or
+
+```sh
+sudo systemctl stop docker
+```
+
+or it may be something else.
+
+#### Run flannel
+
+Now run flanneld itself:
+```sh
+sudo docker -H unix:///var/run/docker-bootstrap.sock run -d --net=host --privileged -v /dev/net:/dev/net quay.io/coreos/flannel:0.3.0
+```
+
+The previous command should have printed a really long hash, copy this hash.
+
+Now get the subnet settings from flannel:
+```
+sudo docker -H unix:///var/run/docker-bootstrap.sock exec <really-long-hash-from-above-here> cat /run/flannel/subnet.env
+```
+
+#### Edit the docker configuration
+You now need to edit the docker configuration to activate new flags.  Again, this is system specific.
+
+This may be in ```/etc/docker/default``` or ```/etc/systemd/service/docker.service``` or it may be elsewhere.
+
+Regardless, you need to add the following to the docker comamnd line:
+```sh
+--bip=${FLANNEL_SUBNET} --mtu=${FLANNEL_MTU}
+```
+
+#### Remove the existing Docker bridge
+Docker creates a bridge named ```docker0``` by default.  You need to remove this:
+
+```sh
+sudo /sbin/ifconfig docker0 down
+sudo brctl delbr docker0
+```
+
+You may need to install the ```bridge-utils``` package for the ```brctl``` binary.
+
+#### Restart Docker
+Again this is system dependent, it may be:
+
+```sh
+sudo /etc/init.d/docker start
+```
+
+it may be:
+```sh
+systemctl start docker
+```
+
+### Starting the Kubernetes Master
```

Comment:
Maybe we should call out that the "main" docker is being used to prevent confusion.